sample-app/src/components/cards/StandardCard.tsx

```diff
@@ -14,28 +47,62 @@ export interface StandardCardProps extends CardProps {
  * @param props - An object containing the result itself.
  */
 export function StandardCard(props: StandardCardProps): JSX.Element {
-  const { configuration, result } = props;
+  const { configuration, result, customCssClasses, compositionMethod } = props;
+  const cssClasses = useComposedCssClasses(builtInCssClasses, customCssClasses, compositionMethod);
 
-  return (
-    <div className='StandardCard'>
-      {configuration.showOrdinal && result.index && renderOrdinal(result.index)}
-      <div className='StandardCard__contentWrapper'>
-        {result.name && renderTitle(result.name)}
+  const cta1 = result.rawData.c_primaryCTA as CtaData;
+  const cta2 = result.rawData.c_secondaryCTA as CtaData;
+
+  // TODO (cea2aj) We need to handle the various linkType so these CTAs are clickable
+  function renderCTAs(cta1?: CtaData, cta2?: CtaData) {
+    return (<>
+      {(cta1 ?? cta2) && 
+        <div className={cssClasses.ctaContainer}>
+          {cta1 && renderFirstCTA(cta1)}
+          {cta2 && renderSecondCTA(cta2)}
+        </div>
+      }
+    </>);
+  }
+
+  function renderFirstCTA({ label, link, linkType }: CtaData) {
+    return (
```

Comment:
nit: probably don't need parentheses if it's just a single HTML element?

Reply:
these functions are so small, so i'll just move the rendering into the `renderCTAs` function